build/automation/var/profile/dev.mk

```diff
@@ -14,9 +14,9 @@ API_DB_USERNAME = postgres
 API_LOG_LEVEL = INFO
 
 #DOS_DB_HOST = [secret]
-DOS_DB_NAME = pathwaysdos_test
+DOS_DB_NAME = postgres
```

Comment:
What's the name of the database of DoS? Can we use it instead of `postgres`?

Reply:
Let's think about this in a different task. I'll add to the TO_DO list.

Reply:
later is fine